code/iaas/logic/src/main/java/io/cattle/platform/process/containerevent/ContainerEventCreate.java

```diff
@@ -159,10 +199,10 @@ void setImage(Map<String, Object> inspect, Instance instance) {
         String imageUuid = (String)config.get(INSPECT_IMAGE);
 
         // Somewhat of a hack, but needed for testing against sim contexts
-        if ( !imageUuid.matches(IMAGE_KIND_PATTERN) ) {
+        if (!imageUuid.matches(IMAGE_KIND_PATTERN)) {
             imageUuid = IMAGE_PREFIX + imageUuid;
         }
-        DataAccessor.fields(instance).withKey(InstanceConstants.FIELD_IMAGE_UUID).set(imageUuid);
+        DataAccessor.fields(instance).withKey(FIELD_IMAGE_UUID).set(imageUuid);
         Image image = null;
         try {
             image = storageService.registerRemoteImage(imageUuid);
```

Comment:
This is all outside the scope of native docker.  I'll take up this work when I do one network per account.  You can merge as is.